### PR TITLE
[1LP][RFR]Fix tree navigation in test_myservice.py tests

### PR DESCRIPTION
**Pull request by rlbabyuk**
Fixed tree navigation, edited path label for finding 'All services' for 5.8 , and also add catalog step to find service in the tree.   

{{ pytest: cfme/tests/services/test_myservice.py -k test_download_file --long-running}}

### CODE REVIEW

File: cfme/services/myservice.py [hunk 1]
```diff
@@ -202,20 +202,27 @@ def download_file(self, extension):
         navigate_to(self, 'All')
         down_btn("Download as " + extension)
 
+    @property
+    def label_all_services(self):
```
Comment (jkrocil) [inline cfme/services/myservice.py]: Let's change this to `all_services_label` instead, please. Method names that start with a verb suggest action so it is not ideal for a property.

**Comment by rlbabyuk** [comment]
Yes, sure, updated

File: cfme/services/myservice.py (hunk 1 above)
Comment (rlbabyuk) [inline cfme/services/myservice.py]: Yes, sure, updated

**Comment by jkrocil** [comment]
@rlbabyuk  `git commit -S --amend`

**Comment by dajoRH** [comment]
Lint report for commit c4af85ec19d7f3c44398286e62fc83ba0d4f8c15:

Lint OK! :cake: :punch: :cookie: .

* c4af85ec19d7f3c44398286e62fc83ba0d4f8c15
  * Merge commit detected!!! :bangbang:

**The PR might not be acceptable in this state** :rage4:.
There is a merge commit in the branch which we do not like in PRs so please, get rid of it.

*CFME QE Bot*

**Comment by dajoRH** [comment]
Would you mind rebasing this Pull Request against latest master, please? :trollface:
*CFME QE Bot*

**Comment by dajoRH** [comment]
Lint report for commit bd10f13ffd6a880badd8b39d41ff85bd0d9ba310:

Lint OK! :cake: :punch: :cookie: .

* bd10f13ffd6a880badd8b39d41ff85bd0d9ba310
  * Merge commit detected!!! :bangbang:

**The PR might not be acceptable in this state** :rage4:.
There is a merge commit in the branch which we do not like in PRs so please, get rid of it.

*CFME QE Bot*

File: cfme/services/catalogs/service_catalogs.py [hunk 2]
```diff
@@ -135,7 +135,7 @@ class ServiceCatalogDetails(CFMENavigateStep):
     def step(self):
         try:
             self.prerequisite_view.service_catalogs.tree.click_path("All Services",
-                self.obj.catalog.name, self.obj.name)
+                    self.obj.catalog, self.obj.name)
```
Comment (psav) [inline cfme/services/catalogs/service_catalogs.py]: The catalog should be an object, why are we removing the name attribute here?

Comment (psav) [inline cfme/services/catalogs/service_catalogs.py]: Especially as we seem to add it in later in the PR

Comment (rlbabyuk) [inline cfme/services/catalogs/service_catalogs.py]: I see what you mean, better to bypass the catalog object and then use attributes. 
Will fix this

**Pull request merged**